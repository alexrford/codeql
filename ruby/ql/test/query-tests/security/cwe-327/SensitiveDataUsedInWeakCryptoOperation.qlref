queries/security/cwe-327/SensitiveDataUsedInWeakCryptoOperation.ql