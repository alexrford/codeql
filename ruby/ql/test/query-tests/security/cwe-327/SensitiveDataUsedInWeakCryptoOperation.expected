edges
| sensitive_data.rb:18:5:18:14 | call to secret_key :  | sensitive_data.rb:23:13:23:13 | k |
| sensitive_data.rb:18:5:18:14 | call to secret_key :  | sensitive_data.rb:30:13:30:13 | k |
nodes
| sensitive_data.rb:13:13:13:24 | call to get_password | semmle.label | call to get_password |
| sensitive_data.rb:15:13:15:22 | call to secret_key | semmle.label | call to secret_key |
| sensitive_data.rb:16:9:16:18 | call to secret_key | semmle.label | call to secret_key |
| sensitive_data.rb:18:5:18:14 | call to secret_key :  | semmle.label | call to secret_key :  |
| sensitive_data.rb:23:13:23:13 | k | semmle.label | k |
| sensitive_data.rb:30:13:30:13 | k | semmle.label | k |
subpaths
#select
| sensitive_data.rb:15:13:15:22 | call to secret_key | sensitive_data.rb:15:13:15:22 | call to secret_key | sensitive_data.rb:15:13:15:22 | call to secret_key | A broken or weak cryptographic algorithm depends on $@. | sensitive_data.rb:15:13:15:22 | call to secret_key | sensitive data froma call to secret_key |
| sensitive_data.rb:16:9:16:18 | call to secret_key | sensitive_data.rb:16:9:16:18 | call to secret_key | sensitive_data.rb:16:9:16:18 | call to secret_key | A broken or weak cryptographic algorithm depends on $@. | sensitive_data.rb:16:9:16:18 | call to secret_key | sensitive data froma call to secret_key |
| sensitive_data.rb:23:13:23:13 | k | sensitive_data.rb:18:5:18:14 | call to secret_key :  | sensitive_data.rb:23:13:23:13 | k | A broken or weak cryptographic algorithm depends on $@. | sensitive_data.rb:18:5:18:14 | call to secret_key | sensitive data froma call to secret_key |
| sensitive_data.rb:30:13:30:13 | k | sensitive_data.rb:18:5:18:14 | call to secret_key :  | sensitive_data.rb:30:13:30:13 | k | A broken or weak cryptographic algorithm depends on $@. | sensitive_data.rb:18:5:18:14 | call to secret_key | sensitive data froma call to secret_key |
